test/BurnableMintableCappedERC20.js

```diff
@@ -318,7 +318,8 @@ describe('BurnableMintableCappedERC20', () => {
                 .to.emit(token, 'Transfer')
                 .withArgs(ethers.constants.AddressZero, user.address, amount);
 
-            await token.connect(user).approve(owner.address, MaxUint256);
+            const txApproval = await token.connect(user).approve(owner.address, MaxUint256);
+            await txApproval.wait();
```

Comment:
```suggestion
            await token.connect(user).approve(owner.address, MaxUint256).then((tx) => tx.wait());
```
tx response isn't needed later, so we can keep it concise